drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/editor/strategies/AbstractDataManagementStrategyTest.java

```diff
@@ -70,6 +72,23 @@ public void getPropertiesToHideListPropertyAssigned() {
         commonGetPropertiesToHideList(true);
     }
 
+    @Test
+    public void getPropertiesToHideListWithSelectedColumnNotInGridModel() {
+        doReturn(true).when(gridColumnMock).isPropertyAssigned();
+        List<String> retrieved = abstractDataManagementStrategySpy.getPropertiesToHide(scenarioGridColumnMock, scenarioGridModelMock);
+        assertTrue(retrieved.isEmpty());
+        reset(scenarioGridModelMock);
+    }
+
+    @Test
+    public void getPropertiesToHideListWithSelectedColumnIsNull() {
+        doReturn(true).when(gridColumnMock).isPropertyAssigned();
+        List<String> retrieved = abstractDataManagementStrategySpy.getPropertiesToHide(null, scenarioGridModelMock);
+        assertTrue(retrieved.isEmpty());
+        verify(scenarioGridModelMock, never()).getSimulation();
+        reset(scenarioGridModelMock);
```

Comment:
No need to reset a mock at the end of the test

Reply:
@yesamer 
About the above statement, remember that soon before execution of a "@Test" annotated method, the "@Before" annotated one is invoked. Reset is only needed when making/verifying multiple "tests" inside the same "@Test" method

Reply:
Test removed